### PR TITLE
remove reporting of tab migration

### PR DESCRIPTION
**Pull request by ecotopian**

### CODE REVIEW

File: Client/Ecosia/Migration/EcosiaImport.swift [hunk 1]
```diff
@@ -36,56 +29,34 @@ final class EcosiaImport {
...
-        return (tabsProgress + favsProgress + historyProgress) / 3.0
+        return (favsProgress + historyProgress) / 2.0
     }
 
     func migrate(progress: ((Double) -> ())? = nil, finished: @escaping (Migration) -> ()) {
-        self.finished = finished
         self.progress = progress
 
-        // Tab migration is triggered by Browser setup in parallel, so it can be it has finished before we get here
-        let ecosiaTabs = Core.Tabs().items
```
Comment (vauxhall) [inline Client/Ecosia/Migration/EcosiaImport.swift:67]: We are not migrating tabs anymore?

Comment (ecotopian) [reply Client/Ecosia/Migration/EcosiaImport.swift:67]: we do - but it's happening outside of this migration abstraction. This code here was only used to monitor the tab migration but the monitoring does not work as expected, so let's remove it. 

Let me point you where the tab migration is happening.
Here it is triggered:
https://github.com/ecosia/ios-browser/blob/3c1ea1197b2db3acef96e2bb6887e22be1947971/Client/Frontend/Browser/TabManagerStore.swift#L22

There it migrates:
https://github.com/ecosia/ios-browser/blob/3c1ea1197b2db3acef96e2bb6887e22be1947971/Client/Frontend/Browser/TabManagerStore.swift#L162

It "just" hooks into the original Tab restoration mechanics which has not produced any errors for the first batches of migrations and therefore needsno extra monitoring any more, agree?

Comment (vauxhall) [reply Client/Ecosia/Migration/EcosiaImport.swift:67]: Thanks for clarifying. Looks neat 💯

**Review by vauxhall — APPROVED** [review]
💯

**Pull request merged**